data/core/units/monsters/Ant_Firebomb.cfg

```diff
@@ -0,0 +1,109 @@
+#textdomain wesnoth-units
+
+#define PATH_TEMP
+units/monsters/ant/#enddef
+
+#define STATIC_IMAGE_TEMP
+{PATH_TEMP}firebag-ant.png~BLIT({PATH_TEMP}fire-ant-glow.png,0,0)#enddef
+
+[unit_type]
+    id=Firebomb Ant
+    name= _ "Firebomb Ant"
+    race=monster
+#ifndef __WMLUNITS__
+    image="{STATIC_IMAGE_TEMP}"
+#else
+    # Shown on units.wesnoth.org instead,
+    # because Image Path Functions aren't available there.
+    image="{PATH_TEMP}firebag-ant.png"
+#endif
+    profile="portraits/monsters/firebag_ant.webp"
```

Comment:
There's a firebomb_ant.webp, but not this.

Reply:
That was the first name, seemed too colloquial.